packages/core/src/components/tree/tree.tsx

```diff
@@ -63,6 +67,14 @@ export class Tree<T = {}> extends React.Component<ITreeProps<T>, {}> {
 
     private nodeRefs: { [nodeId: string]: HTMLElement } = {};
 
+    public nodeFromPath(path: number[]): ITreeNode<T> {
```

Comment:
i'm confused, where was this function defined before?

Reply:
also please put this after `render()`. render should always be the first public method in a react component.

Reply:
This function was not defined before, I just made it as a typed alternative to the static nodeFromPath. I can remove it if you don't think it's necessary though.

Reply:
oh yes please remove. the instance method is far less preferable than the static one because it requires grabbing a ref.